scoring.js

```diff
@@ -0,0 +1,37 @@
+function calculateScore(player) {
+  switch (player.position) {
+    case 'QB':
+      return (
+        (player.stats.passing.touchdowns * 6) +
+        (player.stats.passing.yards / 25) -
+        (player.stats.passing.interceptions * 3) +
```

Comment:
This could be done more efficient.  Don't forget DRY don't repeat yourself.  There could be small calculate functions such as calculate passing, rushing, receiving and returns and then used as needed by position.